index.d.ts

```diff
@@ -5,7 +5,9 @@ type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>
 
 export interface TippyProps extends Omit<Props, 'content'> {
   content: React.ReactElement<any> | string
-  children: React.ReactElement<any>
+  children: (
+    ref: React.RefObject
+  ) => React.ReactElement<any> | React.ReactElement<any>
```

Comment:
Think you need some more parentheses.

```
children: ((
    ref: React.RefObject
  ) => React.ReactElement<any>) | React.ReactElement<any>
```

Reply:
Prettier seems to change that to:

```ts
children:
  | (ref: React.RefObject) => React.ReactElement<any>
  | React.ReactElement<any>
```

Is that valid?